nautobot/core/api/views.py

```diff
@@ -702,3 +709,90 @@ def execute_graphql_request(self, request, data, query, variables, operation_nam
             return document.execute(**options)
         except Exception as e:
             return ExecutionResult(errors=[e], invalid=True)
+
+
+#
+# Lookup Expr
+#
+
+
+class GetFilterSetFieldLookupExpressionChoicesAPI(NautobotAPIVersionMixin, APIView):
+    """API View that gets all lookup expression choices for a FilterSet field."""
+
+    permission_classes = [IsAuthenticated]
+
+    @extend_schema(exclude=True)
+    def get(self, request):
+        if "content_type" not in request.GET or "field_name" not in request.GET:
+            return Response(
+                "content_type and field_name are required parameters",
+                status=400,
+            )
+        contenttype = request.GET.get("content_type")
+        field_name = request.GET.get("field_name")
+        app_label, model_name = contenttype.split(".")
+        try:
+            model = ContentType.objects.get(app_label=app_label, model=model_name).model_class()
```

Comment:
IIRC `model_class` can potentially be `None` in the case where it's a leftover content-type from a plugin that's no longer installed. Probably need to handle that.